roles/nginx/files/production/nginx/sites-available/lobste.rs

```diff
@@ -66,6 +66,14 @@ server {
     break;
   }
 
+  location ~ ^/(apple-touch-icon|touch-icon).*\.png$ {
```

Comment:
Do you want `favicon.ico` covered too? I noticed it also lacks `Cache-Control` headers. Left it off here 'cause you didn't explicitly mention it in the ticket, though.